include/sensors/core/tasks/tip_presence_notification_task.hpp

```diff
@@ -38,12 +41,14 @@ class TipPresenceNotificationHandler {
             can::messages::PushTipPresenceNotification{
                 .message_index = m.message_index,
                 .ejector_flag_status =
-                    static_cast<uint8_t>(hardware.check_tip_presence())});
+                    static_cast<uint8_t>(hardware.check_tip_presence()),
+                .sensor_id = this->sensor_id});
```

Comment:
```suggestion
                .sensor_id = sensor_id});
```

keep it consistent with the rest of our code in general and `visit(TipStatusChangeDetected)` in particular